components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -800,6 +800,13 @@ impl CanvasRenderingContext2DMethods for CanvasRenderingContext2D {
         receiver.recv().unwrap()
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-context-2d-filltext
+    fn FillText(&self, text: DOMString, x: f64, y: f64, max_width: Option<f64>) {
+        let parsed_text: String = text.parse().unwrap();
```

Comment:
This doesn't need `parse()`, you should be able to do `into()`, or if it doesn't exist, you may as well add it.